Point-of-No-Return/Object.h

```diff
@@ -6,6 +6,29 @@
 #include <vector>
 #include <string>
 
+/**
+ * @brief 自身のX座標の値と引数の値を比べる
+ * @param lhs 自身のX座標
+ * @param rhs 比べる値
+ * return 引数の値が大きかったらtureを返す
```

Comment:
typoでしょうね。昨日言えばよかったね。すみません。
`CoordinateY` の方も同様です。
```suggestion
 * return 引数の値が大きかったらtrueを返す
```

Reply:
18b3f67 で修正しました。